### PR TITLE
NXBT-2615: Unicode file

### PR DESCRIPTION
**Pull request by alexistimic**

### CODE REVIEW

**Comment by nelsonsilva** [comment]
👍

File: roles/slave_tools/tasks/nodejs.yml [hunk 1]
```diff
@@ -19,6 +19,13 @@
...
     PATH: /opt/build/tools/node-v4.5.0-linux-x64/bin:{{ansible_env.PATH}}
+# Bypass unstable unicode.org
+- file: path=/usr/share/unicode-data state=directory
+  tags: nodejs
+- copy: src=UnicodeData.txt dest=/usr/share/unicode-data/UnicodeData.txt mode=0644 owner=root group=root
+  tags: nodejs
+- name: Install Node4 dependencies
+  npm: name={{item.name}} version={{item.version}} global=yes
 - name: Install Node4 dependencies
   npm: name={{item.name}} version={{item.version}} global=yes
```
Comment (efge) [inline roles/slave_tools/tasks/nodejs.yml]: These two lines now seem to be duplicated. @alexistimic

**Comment by alexistimic** [comment]
UnicodeData.txt from : ftp://ftp.unicode.org/Public/UNIDATA/UnicodeData.txt

File: roles/slave_tools/tasks/nodejs.yml (hunk 1 above)
Comment (alexistimic) [reply roles/slave_tools/tasks/nodejs.yml]: Yes, unfortunately branch is merged to 9.10
I have commited again on 9.10 to fix

**Pull request merged**